bin/content-push

```diff
@@ -5,6 +5,9 @@ require 'cdo/git_utils'
 require 'cdo/only_one'
 
 CONTENT_PATHS = 'dashboard pegasus aws/dms'.freeze
+LEVELBUILDER_OMISSIONS = [
+  'dashboard/db/schema_cache.dump',
+]
```

Comment:
Please add a brief comment indicating that changes to the files in this list will be reverted during the content push, not just ignored.

Reply:
I was using `git reset` to just ignore them, but realized that would still create problems during the DTL step if there are `schema_cache.dump` changes. I added a `git checkout` to revert the changes, and comment to indicate that.

Reply:
oops, I misread your git command, and was attempting to ask you to at least comment this rather harsh behavior. however you raise a good point that `checkout` is probably best here to prevent DTL problems, so what you have now looks good.